src/wlapplication_options.cc

```diff
@@ -159,365 +186,373 @@ static inline SDL_Keysym keysym(const SDL_Keycode c, uint16_t mod = 0) {
                                                             keysym(SDLK_c, kDefaultCtrlModifier),
                                                             "copy",
-                                                            []() { return _("Copy Text"); })},
+                                                            gettext_noop("Copy Text"))},
    {KeyboardShortcut::kCommonTextPaste, KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kGlobal},
                                                              keysym(SDLK_v, kDefaultCtrlModifier),
                                                              "paste",
-                                                             []() { return _("Paste Text"); })},
+                                                             gettext_noop("Paste Text"))},
    {KeyboardShortcut::kCommonSelectAll, KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kGlobal},
                                                              keysym(SDLK_a, kDefaultCtrlModifier),
                                                              "selectall",
-                                                             []() { return _("Select All"); })},
+                                                             gettext_noop("Select All"))},
    {KeyboardShortcut::kCommonDeleteItem,
     KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kGlobal},
                          keysym(SDLK_DELETE),
                          "delete",
-                         []() { return _("Delete Item"); })},
+                         gettext_noop("Delete Item"))},
    {KeyboardShortcut::kCommonTooltipAccessibilityMode,
     KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kGlobal},
                          keysym(SDLK_F2),
                          "tt_access_mode",
-                         []() { return _("Tooltip Accessibility Mode Key"); })},
+                         gettext_noop("Tooltip Accessibility Mode Key"))},
    {KeyboardShortcut::kCommonFullscreen,
     KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kGlobal},
                          keysym(SDLK_f, kDefaultCtrlModifier),
                          "fullscreen",
-                         []() { return _("Toggle Fullscreen"); })},
+                         gettext_noop("Toggle Fullscreen"))},
    {KeyboardShortcut::kCommonScreenshot,
     KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kGlobal},
                          keysym(SDLK_F11),
                          "screenshot",
-                         []() { return _("Take Screenshot"); })},
+                         gettext_noop("Take Screenshot"))},
+   {KeyboardShortcut::kCommonDebugConsole,
+    KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kGlobal},
+                         keysym(SDLK_SPACE, kDefaultCtrlModifier | KMOD_SHIFT),
+                         "debugconsole",
+                         gettext_noop("Open the Debug Console (only in debug builds)"))},
+   {KeyboardShortcut::kCommonCheatMode,
```

Comment:
> These only work in release builds by design,

I thought they are working only in debug builds, not sure about the cheat mode though. If possible they should be shown in debug builds, but not in release builds, imho.

Reply:
Well, having them here makes sure that no conflicting shortcut can be defined. (of course it doesn't hurt either to have them configurable) This is important, because these are evaluated very early in the `handle_key` chain.

If I make them `#ifndef NDEBUG`, then they will be dropped from the config if a release build is run with the same config dir, and again, conflicting shortcuts could be defined.

I could put them in a new region and only show them in the help in debug builds, but still have them in options.

Or, how about allowing the script console in release builds too with a command line option (e.g. `--script_debugging`)? (I could make it part of solving #5793) Why force content developers to use dead slow debug builds when they only want to fiddle with their scripts? :)

Reply:
@Noordfrees We had some more proof of demand for the script console in release builds recently: https://github.com/widelands/widelands/issues/5852#issue-1667142245:

> The only thing I want from the "Debug" build is the in-game script console. I do not want debugging symbols compiled into the code.

Reply:
Well, then how about making the scripting console available in `RelWithDebInfo` builds as well via a `-Ddefine`. All the benefits of release builds plus additional debugging utilities, of which the scripting console, debug window, and cheating mode are three. Pure release builds really shouldn't have those debugging-only features though IMO.

Reply:
@Noordfrees: Would you accept for this PR:

Put the shortcuts for script console and cheat mode in a new region and only show them in the help in debug builds, but still have them configurable in options.